security/pkg/pki/ca/controller/secret.go

```diff
@@ -269,7 +273,7 @@ func (sc *SecretController) scrtUpdated(oldObj, newObj interface{}) {
 	// to expire, or 2) the root certificate in the secret is different than the
 	// one held by the ca (this may happen when the CA is restarted and
 	// a new self-signed CA cert is generated).
-	if ttl.Seconds() < secretResyncPeriod.Seconds() || !bytes.Equal(rootCertificate, scrt.Data[RootCertID]) {
```

Comment:
Can we delete secretResyncPeriod as well?

Reply:
It's also used for the cache informer, so it makes sense to keep it :)